velox/docs/functions/spark/decimal.rst

```diff
@@ -27,14 +27,28 @@ Multiplication
 
 Division
 --------
-
+When config ``SparkRegistrationConfig.allowPrecisionLoss`` is set to true as default.
 ::
 
     p = p1 - s1 + s2 + max(6, s1 + p2 + 1)
     s = max(6, s1 + p2 + 1)
 
+When config ``SparkRegistrationConfig.allowPrecisionLoss`` is set to false.
+::
+
+    intDig = min(38, p1 - s1 + s2);
+    decDig = min(38, max(6, s1 + p2 + 1));
+If ``indDig + decDig`` is more than 38:
+::
+    p = 38
+    s = decDig - (intDig + decDig - 38) / 2 - 1
+Otherwise:
+::
+    p = intDig + decDig
+    s = decDig
+
 For above arithmetic operators, when the precision of result exceeds 38,
```

Comment:
This part applies for add/subtract/multiply/divide instead of just divide, to make this clear, could you help add a sub-title like 'Decimal Precision and Scale Adjustment'?